From 9c167914a1d88192882fe09dfce430a299580a8a Mon Sep 17 00:00:00 2001
From: Dave Yeo <dave.r.yeo@gmail.com>
Date: Tue, 30 Oct 2012 23:48:26 -0700
Subject: [PATCH 097/278] x86: Fix assembly with NASM

Unlike YASM, NASM only looks for include files in the current
directory, not in the directory that included files reside in.

Signed-off-by: Diego Biurrun <diego@biurrun.de>
---
 libavutil/x86/x86util.asm |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavutil/x86/x86util.asm b/libavutil/x86/x86util.asm
index d7ec594..4c6f4c6 100644
--- a/libavutil/x86/x86util.asm
+++ b/libavutil/x86/x86util.asm
@@ -23,7 +23,7 @@
 ;* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 ;******************************************************************************
 
-%include "x86inc.asm"
+%include "libavutil/x86/x86inc.asm"
 
 %macro SBUTTERFLY 4
 %if avx_enabled == 0
-- 
1.7.9.5

